docs/next/guides/cell-functions/cell-validator.md

```diff
@@ -188,12 +188,13 @@ Edit the above grid to see callback
 
 ## Related articles
 
+::: only-for javascript
 ### Related guides
 
-- [Custom renderer in React](@/guides/integrate-with-react/react-custom-renderer-example.md)
 - [Custom renderer in Angular](@/guides/integrate-with-angular/angular-custom-renderer-example.md)
 - [Custom renderer in Vue 2](@/guides/integrate-with-vue/vue-custom-renderer-example.md)
 - [Custom renderer in Vue 3](@/guides/integrate-with-vue3/vue3-custom-renderer-example.md)
+:::
```

Comment:
This is mind-bending... I don't think that "Custom renderer" is a related article to "Cell validator". We should remove the "Related guides" from this page completely.

Reply:
Done: [af12e2f](https://github.com/handsontable/handsontable/pull/9576/commits/af12e2f3e6ab446222958e6f6d5f03467c76c483)